server/pkg/item/field_change.go

```diff
@@ -0,0 +1,85 @@
+package item
+
+
+type FieldChangeType string
+
+const (
+	Add    FieldChangeType = "add"
+	Update FieldChangeType = "update"
+	Delete FieldChangeType = "delete"
+)
+
+
+type FieldChange struct {
+	Id *FieldID
```

Comment:
```suggestion
	ID *FieldID
```

In golang, abbreviations that are normally written in capital letters should be left as they are.

Reply:
@rot1024 , I changed it, but it represents the hook return data so now the hook returns:

```
 "fieldschange": [
      {
        "CurrentValue": [
          "added value"
        ],
        "ID": "01h5v15gfkrerr0zrqa914zan6",
        "PreviousValue": null,
        "Type": "add"
      }
    ],
```

Reply:
```go
type A struct {
	ID string `json:"id"`
}
```

You can use "tag". For details, google it